docs/reference/commandline/dockerd.md

```diff
@@ -930,7 +930,7 @@ This is a full example of the allowed configuration options in the file:
 	"ipv6": false,
 	"iptables": false,
 	"ip-forward": false,
-	"ip-mask": false,
+	"ip-masq": false,
```

Comment:
Hm, I think this needs more than a documentation change; and we may need to accept both; see https://github.com/docker/docker/blob/28b7dc51aec6a237af187badfc1953eb9dbbb142/daemon/config_unix.go#L44